aten/src/ATen/native/native_functions.yaml

```diff
@@ -9068,6 +9068,11 @@
   dispatch:
     CPU, CUDA: glu_jvp
 
+- func: glu_vjp(Tensor grad_x, Tensor grad_glu, Tensor x, Tensor dgrad_glu, Tensor dx, int dim) -> Tensor
```

Comment:
Oh whoops I missed this in the previous PR for `glu_jvp`, but I think we should add ".*jvp" in `_SKIP_PYTHON_BINDINGS` in `gen_python_functions` (this is already done for `_backward`).

Reply:
Will do.